x/bank/simulation/genesis.go

```diff
@@ -35,6 +35,15 @@ func RandomGenesisSendParams(r *rand.Rand) types.SendEnabledParams {
 	return params.SendEnabled
 }
 
+func GenesisCoins(bondDenom string, initialStake int64) sdk.Coins {
+	coins := make(sdk.Coins, 20)
```

Comment:
```suggestion
	coins := make(sdk.Coins, 8)
```
I suspect the reason the simulation tests took so much longer was because of the high number here